tests/test_asyncio.py

```diff
@@ -196,6 +196,7 @@ async def test_connect_and_serve_with_ed25519_certificate(self):
             )
         )
 
+    @skipIf("ed448" in SKIP_TESTS, "Skipping ed448 tests")
```

Comment:
Where is this used?

Reply:
I think I don't understand your question. Can I ask you to rephase it?